en/components/list-skeleton.md

```diff
@@ -47,12 +47,20 @@ There are 4 interchangeable List Item Primary Actions like the standard [List](l
 
 ## List Item Secondary Action
 
-The Secondary Action of the Skeleton List Item covers all scenarios possible with its counterpart from the standard [List](list.md). Thanks to the use of Smart Layout in Sketch it can be adjusted further by setting unnecessary elements to ~No Symbol.
+In Sketch, The Secondary Action of the Skeleton List Item covers all scenarios possible with its counterpart from the standard [List Item](list-item.md) available as `Symbol Overrides`. Thanks to Smart Layout in Sketch, it can be adjusted further by setting unnecessary elements to ~No Symbol.
```

Comment:
```suggestion
In Sketch, The Secondary Action of the Skeleton List Item covers all scenarios possible with the variety of options in the standard [List Item](list-item.md). Thanks to Smart Layout in Sketch, it can be adjusted further by setting unnecessary elements to ~No Symbol.
```